src/components/NewCommentForm/NewCommentForm.scss

```diff
@@ -0,0 +1,19 @@
+.NewCommentForm {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+
+  &__input {
+    border: none;
+    border-bottom: 1px solid #4d457b;
+    font-family: inherit;
+    font-size: 0.9rem;
+    padding: 10px 5px;
+    color: #4d457b;
```

Comment:
Use `scss` variables; you are using this color in several places, so it makes sense to create a separate var